flink-test-utils-parent/flink-test-utils/pom.xml

```diff
@@ -115,6 +115,14 @@ under the License.
 			</exclusions>
 		</dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-statebackend-changelog</artifactId>
+            <version>${project.version}</version>
+            <!-- the dependency should be transitive -->
+            <scope>runtime</scope>
```

Comment:
As discussed in the thread, `compile` is suggested here?

Reply:
It looks like no strong position was expressed on `runtime` vs `compile` in that discussion.

To me, `compile` is a bit superflous here as it allows to use backend code in `flink-test-utils` explicitly; 
while for dependants, both `compile` and `runtime` [result](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope) in the same `test` scope.

WDYT? 

cc: @zentol

Reply:
Yes, that makes sense to me. I also think `runtime` is enough if purely for dependants.

But all other dependencies put in `flink-test-utils` are in `compile` level, that's why I am asking.

cc @zentol . Could you please double confirm?

I have no strong opinion here.

Reply:
I would go with `runtime` because we want neither flink-test-utils nor other modules to reference classes from the changelog backend in production code (which `runtime` prevents).
We really just want to put it on the test classpath.

Reply:
However please double-check that no module is now accidentally bundling the changelog backend.

Reply:
Thanks, good point. Only fat jars under `flink-end-to-end-tests` now contain the ChangelogStateBackend classes (like  HeavyDeploymentStressTestProgram.jar or KinesisExample.jar).